### PR TITLE
Wip mega task for migrating

### PR DESCRIPTION
**Pull request by gigxz**
needs cleanup, but here's what i've got @ttoomey

### CODE REVIEW

File: drivers/hmis_external_apis/lib/tasks/import.rake [hunk 1]
```diff
@@ -19,4 +19,73 @@ namespace :import do
...
+
+    # Pull down CSV
+    today = Date.current.strftime('%Y-%m-%d')
+    dir = "/tmp/migration/#{today}/custom-data"
+    FileUtils.mkdir_p dir
+    creds = GrdaWarehouse::RemoteCredentials::S3.find_by(slug: HmisExternalApis::AcHmis::Importers::S3ZipFilesImporter::MPER_SLUG)
+    s3 = creds.s3
+    s3.list_objects(prefix: 'initial-migration')
+    zipfile = "#{dir}/custom.zip"
+    s3.fetch(file_name: '2023-08-18-HMIS-Legacy-Data-csv-a28b08c4-3098-4d34-8575-d5896a018f6c-.zip', prefix: 'initial-migration', target_path: zipfile)
```
Comment (gigxz) [inline drivers/hmis_external_apis/lib/tasks/import.rake:63]: date should be `today`
```

File: drivers/hmis_external_apis/lib/tasks/import.rake [hunk 2]
```diff
@@ -19,4 +19,133 @@ namespace :import do
...
+    HmisExternalApis::AcHmis::Referral.joins(:postings).group(HmisExternalApis::AcHmis::ReferralPosting.arel_table[:status]).count
+    # The below should be 0. All Accepted/Pending should be tied to a household.
+    HmisExternalApis::AcHmis::ReferralPosting.where(status: ['accepted_status', 'accepted_pending_status']).where(HouseholdID: nil).count
+
+    # Count how many MCI and MCI Unique IDs
+    HmisExternalApis::ExternalId.for_clients.group(:namespace).count
+    # Num Clients wihout MCI Unique IDs
+    Hmis::Hud::Client.hmis.left_outer_joins(:ac_hmis_mci_unique_id).where(ac_hmis_mci_unique_id: { id: nil }).count
+    # Num Clients wihout MCI IDs
+    Hmis::Hud::Client.hmis.left_outer_joins(:ac_hmis_mci_ids).where(ac_hmis_mci_ids: { id: nil }).count
```
Comment (ttoomey) [inline drivers/hmis_external_apis/lib/tasks/import.rake:97]: Looks like these counts aren't used. Maybe they should just be comments?

**Review by ttoomey — APPROVED** [review]
Looks good to me!!

File: drivers/hmis_external_apis/lib/tasks/import.rake (hunk 2 above)
Comment (gigxz) [reply drivers/hmis_external_apis/lib/tasks/import.rake:97]: Yeah definitely.. I want to clean this up a bit more before merging, it's also just a random function in this file

**Pull request closed without merging**